package.json

```diff
@@ -20,7 +20,7 @@
     "test:debug": "npm run test:build && TEST=1 mocha --inspect-brk",
     "prettify": "balena-lint -e ts -e js --typescript --fix src/ test/ typings/ build-utils/",
     "test-build": "tsc --project tsconfig.json",
-    "release": "tsc --project tsconfig.release.json && cp -r build/src/* build && rm -rf build/src",
+    "release": "tsc --project tsconfig.release.json && mv build/src/* build",
```

Comment:
Why have you made this change? We've still got `src/` there.

Reply:
I can remove the empty dir if you want but it won't affect anything.  
As for why mv instead of cp, it's twofold:
1. it works better on windows so running `npx tsc --project tsconfig.release.json; mv build/src/* build` can work whereas `cp ..; rm -rf build/src` won't (the way to pass the `-f` flag is slightly different because with just `-f` it's ambiguous with another flag)
2. it should be faster as it's not having to copy all the contents and can instead change just the file pointers - on some filesystems with the right options passed to cp and the right versions of things then it may do a COW and end up equivalent, but that's a lot of ifs when a mv makes it simple :P

Reply:
No that's fine, it was just a bit of a strange change so wanted further clarification.